code/modules/fishing/sources/source_types.dm

```diff
@@ -481,6 +481,7 @@
 		/obj/item/seeds/random = 1,
 		/mob/living/basic/frog = 1,
 		/mob/living/basic/axolotl = 1,
+		/mob/living/basic/turtle = 2,
```

Comment:
Considering that's roughly a one in 3.8% chance of catching a turtle, I wonder if using a good/great quality bait should increase the chances of catching mobs a little? It should be as simple as adding some if checks to the `get_modified_table` proc.